sources/packages/api/src/route-controllers/institution-locations/institution-locations.controller.service.ts

```diff
@@ -0,0 +1,71 @@
+import { Injectable } from "@nestjs/common";
+import { InstitutionLocationService } from "../../services";
+import {
+  DesignationLocationAgreementStatus,
+  InstitutionLocationsDetailsDto,
+} from "./models/institution-location.dto";
+
+@Injectable()
+export class InstitutionLocationsControllerService {
+  constructor(private readonly locationService: InstitutionLocationService) {}
+
+  /**
+   * Retrieve institution locations and
+   * their associated designated status'.
+   * @param institutionId this value is passed only for client type Institution.
+   * @returns designation institution locations with their status'.
+   */
+  async getInstitutionLocationsWithDesignationStatus(
```

Comment:
I believe designationStatus is a part of institution location summary/detail.  Why is the method name giving emphasis on retrieving with designationStatus?